userspace/uv/pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[project]
+name = "agnos"
+version = "0.1.0"
+authors = [{name = "Vehicle Researcher", email = "user@comma.ai"}]
+requires-python = "~=3.11"
+description = ""
+dependencies = [
+  "openpilot",
+  "openpilot[testing]",
```

Comment:
ideally we want all the openpilot groups

Reply:
"testing" and "dev" make sense. I don't see the point for "docs" though. what do you think?

Reply:
although:
- the "dev" dependencies are not useful for normal users
- the "testing" must be there for testing, but again not useful for normal users 

In the cleanup phase or when we want a minimal image, we can simply keep just "openpilot" and install dev+testing+docs with one command (unminimize).

Reply:
yeah, that's fine as long as we have that unminimize command and it's in the MOTD